File: python-sdk/src/foremapi/models/shared/followedtag.py
import dataclasses
from dataclasses_json import dataclass_json
from foremapi import utils


@dataclass_json
@dataclasses.dataclass
class FollowedTag:
    r"""FollowedTag
    Representation of a followed tag
    """
    
    id: int = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('id') }})
    name: str = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('name') }})
    points: float = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('points') }})
    

File: ts-sdk/src/sdk/models/shared/followedtag.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



// FollowedTag
/** 
 * Representation of a followed tag
**/
export class FollowedTag extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "json, name=id" })
  id: number;

  @SpeakeasyMetadata({ data: "json, name=name" })
  name: string;

  @SpeakeasyMetadata({ data: "json, name=points" })
  points: number;
}
